packages/data-store/src/migrations/3.createPrivateKeyStorage.ts

```diff
@@ -41,7 +39,7 @@ export class CreatePrivateKeyStorage1629293428674 implements MigrationInterface
       true,
     )
     // 2. copy key data
-    const keys: PreMigrationKey[] = await queryRunner.manager.find(PreMigrationKey)
+    const keys: PreMigrationKey[] = await queryRunner.manager.query(`SELECT * FROM key`);
```

Comment:
Why is this something different from what it was?

Reply:
find(PreMigrationKey) is also giving the error that we get when we use TableNameStrings instead of table object. I am not sure if we have any method in QueryRunning, QueryManager that can use a tableObject to byPass the issue hence the simple query was run. PreMigrationKey Entity is using the 'key' named table.

Reply:
Added the note to PR.